TACC/Core-Portal
# Task/fp 877  onboarding admin UI tweaks

## To Do:

- [ ] __After Approval, Before Merge:__
    1. Remove all `/* !!!: Temporary […] */` comments.
    2. Run ESLint.
    3. Ensure changes are non-functional.

## Overview: ##

UI Fixes for PR #349.

## Related Jira tickets: ##

* [FP-877](https://jira.tacc.utexas.edu/browse/FP-877)

## Summary of Changes: ##

See issues in [#349 (review) UI comment by Wes](https://github.com/TACC/Core-Portal/pull/349#pullrequestreview-605696405) that are __both__ marked completed (~~stricken through~~) __and__ labelled for this PR.

## Testing Steps: ##
1. Load this branch.
2. Setup user state as done for #349 testing.
3. Make window small (800px–1000px).
4. Ensure all UI issues, see "Summary of Changes", are fixed.

## UI Photos:

<details><summary>Admin - Wide</summary>

![Admin - Wide](https://user-images.githubusercontent.com/62723358/111374641-59bb4580-866b-11eb-84aa-3f6ae2289fc1.jpeg)

</details>
<details><summary>Admin - (Very) Narrow</summary>

![Admin - Narrow (Extra)](https://user-images.githubusercontent.com/62723358/111374639-5922af00-866b-11eb-8867-c0478ad94ec0.jpeg)

</details>
<details><summary>Admin - (Very Very) Narrow</summary>

_This narrow viewport is shown _only_ to prove truncation works for `<Pill>`s._

![Admin - Narrow (Ultra)](https://user-images.githubusercontent.com/62723358/111378145-79ed0380-866f-11eb-83f6-a320c0df6c1c.jpeg)

> \* See "Known Issue" → "Very Very Narrow Admin Screen has Text Overlap".

</details>

<details><summary>Setup - Wide</summary>

![Setup - Wide](https://user-images.githubusercontent.com/62723358/111374650-5aec7280-866b-11eb-8514-0c4a3d0c6ddd.jpeg)

</details>
<details><summary>Setup - (Very) Narrow</summary>

![Setup - Narrow (Extra)](https://user-images.githubusercontent.com/62723358/111374644-5a53dc00-866b-11eb-9f14-a5d267e987d7.jpeg)

</details>
<details><summary>Setup - (Very Very) Narrow</summary>

![Setup - Narrow (Ultra)](https://user-images.githubusercontent.com/62723358/111374647-5a53dc00-866b-11eb-8b90-52e0ac114975.jpeg)

</details>

## Notes: ##

### Known Issue

- __Very Very Narrow Admin Screen has Text Overlap__: The app is not intended to be usable this narrow, because this is too narrow for an app with no mobile design. After PR #350, we can easily limit the width of this section to prevent user from seeing it this narrow.
